index.qmd

```diff
@@ -62,15 +62,15 @@ TODO: check if the content is still valid after rephrasing it with AI
 for the purpose of this paper:
 
 From their origins in the Renaissance, the natural sciences have been
-deeply intertwined with computation. Early pioneers like Kepler and
+intertwined with computation. Early pioneers like Kepler and
```

Comment:
Here and elsewhere: em-dash is usually typeset without whitespace in academic journals ([Merriam Webster](https://www.merriam-webster.com/grammar/em-dash-en-dash-how-to-use)). Maybe you confused them with the similarly-looking en-dashes, which often have whitespaces, except when used to delimit a range of numbers.